modern-business/functions.php

```diff
@@ -127,3 +127,16 @@ function a8c_fse_insert_theme_template_data() {
 }
 
 add_action( 'after_switch_theme', 'a8c_fse_insert_theme_template_data' );
+
+/**
+ * Add Full Site Editing class to body so block specific styles can be targetted
+ * to theme parts if FSE enabled.
+ */
+add_filter('body_class', 'fse_body_class');
```

Comment:
I'd suggest moving this to a plugin (per https://github.com/Automattic/themes/pull/1211#issuecomment-519074802). Also this section is missing spaces after/before parentheses.

Reply:
Yeh that makes sense, will do that - and will get my IDE php formatting set up properly too :-)